src/components/views/ErrorLogs.js

```diff
@@ -1,30 +1,58 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
-import { Accordion } from '@folio/stripes/components';
+import { get } from 'lodash';
+import { Accordion, Badge, MultiColumnList } from '@folio/stripes/components';
 
 export default class ErrorLogs extends React.Component {
   static propTypes = {
     id: PropTypes.string,
+    job: PropTypes.object,
     onToggle: PropTypes.func,
     open: PropTypes.bool,
   };
 
-  renderErrorLogs = () => {
-    return <FormattedMessage id="ui-local-kb-admin.noErrorLogs" />;
+  renderBadge = () => {
+    const count = get(this.props.job, 'errorLog.length', 0);
+    return <Badge>{count}</Badge>;
   }
 
-  render() {
-    const { id, onToggle, open } = this.props;
+  renderErrorLogs = (job) => {
+    const { errorLog } = job;
+    if (errorLog) {
+      return (
+        <MultiColumnList
+          columnMapping={{
+            recordNumber: <FormattedMessage id="ui-local-kb-admin.columns.recordNumber" />,
+            message: <FormattedMessage id="ui-local-kb-admin.columns.errorLogMessage" />,
+          }}
+          columnWidths={{
+            recordNumber: '10%',
+            message: '90%',
+          }}
+          contentData={errorLog}
+          formatter={{ recordNumber: ({ recordNumber }) => (recordNumber || '-') }}
```

Comment:
I don't know the format of `recordNumber`, but the fact that it's a "number" gives me cause for the following concern: if `recordNumber` is ever legitimately `0`, then you'll render a `-` here instead of the `0` you really should. So `recordNumber !== undefined ? recordNumber : '-'` would be safer.